roslibrust_ros1/src/node/actor.rs

```diff
@@ -197,7 +194,7 @@ impl NodeServerHandle {
 
         let md5sum;
         let md5sum_res =
-            roslibrust_codegen::message_definition_to_md5sum(topic_type, msg_definition);
+            roslibrust_common::md5sum::message_definition_to_md5sum(topic_type, msg_definition);
         match md5sum_res {
```

Comment:
This is an aside, but perhaps:
```rust
let md5sum = roslibrust_common::md5sum::message_definition_to_md5sum(
    topic_type,
    msg_definition
).map_err(|err| {
    log::error!("{err:?}");
    Err(NodeError::IoError(io::ErrorKind::ConnectionAborted.into()))
})?;
```

Reply:
It might be nice to change the name of `message_definition_to_md5sum` to `from_message_definition` since it's already namespaced under `md5sum` module.